src/c/cmake/thirdparty/FindCaliper.cmake

```diff
@@ -0,0 +1,31 @@
+## most common case: caliper is built with adiak support
+## and caliper needs us to find adiak, or else find_pacakge caliper
+## will fail
+#
+## Check for ADIAK_DIR
+#
+#if(NOT ADIAK_DIR)
+#    MESSAGE(FATAL_ERROR "Caliper support needs explicit ADIAK_DIR")
+#endif()
+#
+#message(STATUS "Looking for Adiak in: ${ADIAK_DIR}")
+#
+#find_package(adiak REQUIRED
+#             NO_DEFAULT_PATH
+#             PATHS ${ADIAK_DIR}/lib/cmake/adiak)
```

Comment:
I don't think adiak needs to be a firm requirement to use caliper, but should we explore adiak as part of this PR?

Reply:
Let's leave Adiak for Spencer's PR, let me remove it from this PR @slabasan. Thank you for catching this!